source/concepts/account.rst

```diff
@@ -37,6 +37,19 @@ Address
 Each account has a unique :ref:`address <address>` derived from the public key.
 Normally, the address is shared instead of the public key because it is shorter and gathers information about the network.
 
+Hierarchical Deterministic Wallets
+===========
+
+Symbol supports a multi-account hierarchy enabling the generation of multiple public keys without revealing the private key.
```

Comment:
I would also add at least an explanation of why it's relevant to generate HD wallets (store one set of mnemonic codes instead of multiple account private keys, have the possibility to backup accounts...).

Reply:
https://github.com/nemtech/NIP/blob/master/NIPs/nip-0006.md#motivation